src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -72,11 +72,13 @@ public static Answer parseAnswer(String answer) throws ParseException {
     public static Tag parseTag(String tag) throws ParseException {
         requireNonNull(tag);
         String trimmedTag = tag.trim();
-        // user can't type untagged
-        if (!Tag.isValidTagName(trimmedTag) || trimmedTag.toLowerCase().equals("untagged")) {
+
+        if (!Tag.isValidTagName(trimmedTag)) {
```

Comment:
isValidTagName will check if tagName is either "easy", "medium", "hard". Case-insensitive.